### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crisgarner**
Updated prank, prankStart and prankStop usage documentation

### CODE REVIEW

File: forge/README.md [hunk 1]
```diff
@@ -131,11 +131,15 @@ which implements the following methods:
...
 
-- `function etch(address where, bytes memory what)`:` Sets the contract code at
+- `function etch(address where, bytes memory what)`: Sets the contract code at
   some address contract code
 
-- `function prank(address from, address to, bytes calldata) (bool success,bytes retdata)`:
-  Performs a smart contract call as another address
+- `function prank(address from)`: Performs the next smart contract call as another address
+
+- `function prankStart(address from)`: Performs smart contract calls as another address
```
Comment (gakonst) [inline forge/README.md:139]: they are called startPrank and stopPrank, if you can change that

**Review by gakonst — CHANGES_REQUESTED** [review]

File: forge/README.md (hunk 1 above)
Comment (crisgarner) [reply forge/README.md]: Fixed it, got confused with the PRs old names

File: forge/README.md [hunk 2]
```diff
@@ -131,11 +131,15 @@ which implements the following methods:
 
 - `function deal(address who, uint256 amount)`: Sets an account's balance
 
-- `function etch(address where, bytes memory what)`:` Sets the contract code at
+- `function etch(address where, bytes memory what)`: Sets the contract code at
   some address contract code
 
-- `function prank(address from, address to, bytes calldata) (bool success,bytes retdata)`:
-  Performs a smart contract call as another address
+- `function prank(address from)`: Performs the next smart contract call as another address (prank just changes tx.origin. Tx still occurs as normal)
```
Comment (gakonst) [inline forge/README.md:137]:
```suggestion
- `function prank(address from)`: Performs the next smart contract call as another address (prank just changes msg.sender. Tx still occurs as normal)
```

https://github.com/gakonst/foundry/blob/a6af7b552c5d14aaf5df64836dccc19428e97436/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs#L460

File: forge/README.md [hunk 3]
```diff
@@ -131,11 +131,15 @@ which implements the following methods:
...
+
+- `function startPrank(address from)`: Performs smart contract calls as another address
```
Comment (gakonst) [inline forge/README.md:139]:
```suggestion
- `function startPrank(address from)`: Performs smart contract calls as another address. The account impersonation lasts until the end of the transaction, or until `stopPrank` is called.
```

File: forge/README.md [hunk 4]
```diff
@@ -131,11 +131,15 @@ which implements the following methods:
...
+
+- `function stopPrank()`: Stop calling smart contract with the address set at prankStart
```
Comment (gakonst) [inline forge/README.md:141]:
```suggestion
- `function stopPrank()`: Stop calling smart contracts with the address set at `startPrank`
```

**Review by gakonst — APPROVED** [review]

**Pull request merged**